install.ps1

```diff
@@ -76,6 +76,8 @@ param (
     $ChannelUrl = "https://update.rke2.io/v1-release/channels"
 )
 
+#Requires -RunAsAdministrator
```

Comment:
We already have an admin check [here](https://github.com/rancher/rke2/blob/master/install.ps1#L125-L130)

Reply:
Yeah this doesn't work... The script wont even run at all when no elevated, see the issue this PR is attached to. So the check never happens because the script never gets that far. This requires script is always run regardless of where it is in the file. See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_requires?view=powershell-7.2#rules-for-use